lib/index.ts

```diff
@@ -565,6 +685,14 @@ export namespace TopicRequest {
 }
 
 // Export Types ///////////////////////////////////////////////////////////////
+export type EventCallback = (error: Error | null, event?: any) => void;
+
+export interface EventOptions {
```

Comment:
NIT: I think `EventHookOptions` -- or something to that effect -- is more appropriate.